Formula/tzdiff.rb

```diff
@@ -0,0 +1,17 @@
+class Tzdiff < Formula
+  desc "Displays Timezone differences with localtime in CLI (shell script)"
+  homepage "https://github.com/belgianbeer/tzdiff"
+  url "https://github.com/belgianbeer/tzdiff/archive/1.2.tar.gz"
+  sha256 "6c3b6afc2bb36b001ee11c091144b8d2c451c699b69be605f2b8a4baf1f55d0a"
+  license "BSD-2-Clause"
```

Comment:
I noticed that your repository doesn't contain a `LICENSE` file. Do you mind adding one? Using this file allows GitHub to recognise the license for your repository (and we rely on GitHub doing this for some of our automated checks).

Reply:
I would like LICENSE file added.
However, the license is written directly in both tzdiff and tzdiff.1.

Reply:
The license is written directly to both tzdiff and tzdiff.1. If still needed, I will add a LICENSE file.

Reply:
Yes, it would be nice for you to have a separate LICENSE file in addition to what is already in the source headers.

Reply:
I found the document, "[Licensing a repository](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository)". And, am I correct in understanding that the LICENSE file should be prepared independently of the tzdiff 1.2 release?

Reply:
Good catch, that is right, just create a LICENSE file, it would be bundled with the source code archive..

Just raised a PR for it. https://github.com/belgianbeer/tzdiff/pull/7